xhr.bs

```diff
@@ -424,10 +424,9 @@ and
 method steps are:
 
 <ol>
- <li><p>Let <var>settingsObject</var> be <a>this</a>'s <a>relevant settings object</a>.
-
- <li><p>If <var>settingsObject</var> has a <a>responsible document</a> and it is <em>not</em>
- <a>fully active</a>, then <a>throw</a> an "{{InvalidStateError!!exception}}" {{DOMException}}.
+ <li><p>If the <a>current global object</a> is a {{Window}} object and its
```

Comment:
If you want to translate the previous text precisely, you need to check this's relevant global object. They can be different if doing `otherWindow.XMLHttpRequest.prototype.open.call(thisWindowXHR)`

Reply:
This check matches the other "is Window" checks elsewhere in the specification, but I see now how we don't grab state from them there. Oops.

Reply:
Yeah, I guess when you're just doing checks, the current global object is a Window if and only if this's relevant settings object is a Window.